### ISSUE TITLE
highlight the result

### ISSUE THREAD

**Issue by nico92GitHub** [bleachbit/bleachbit#1473]
Hello Dev

Really, you don't want to highlight the result of a cleaning?
Something that would look like the attachment or this:
**Espace disque récupéré : 18.3MB**
Fichiers supprimés : 138
Opérations spécifiques : 10
elapsed time: 9 seconds
![ScreenShot_001](https://user-images.githubusercontent.com/22394276/224538147-dfe71aaf-dc91-4e6e-bb03-f15be0e502a4.jpg)

We look forward to your return,
best regards,
Nico

**Comment by nico92GitHub** [comment]
Well received and Understood your message.
I really had no intention of harassing you or stealthily taking advantage of the work of volunteer developers.
Not having received an answer, I Just renewed my wish without bad intention.

Nico

**Comment by kenrap** [comment]
Alright. Let me apologize for my rough response. I was concerned that this type of behavior would repeat itself if it gets ignored again.

I don't consider what you did is harassment. It seemed like it came off being pushy by opening a new following issue. I got burnt out quite a few times in the past from contributing to open source projects and that's why I expressed my argument that way.

With that said, I think this feature is a good idea (perhaps as a checkbox option). My point about replying to the original issue with the screenshot is that it would "update" the issue. As in, this makes it easier for bug-hunters to find your issue by sorting through "recently updated" issues. Replying with the screenshot would make your argument seem stronger in my opinion.

Either way, I'm very sorry.
